tests/bluetooth/host/hci/testcase.yaml

```diff
@@ -0,0 +1,14 @@
+tests:
+  bluetooth.host.hci:
+    platform_allow:
+      - native_posix
+      - native_posix/native/64
+      - native_sim
+      - native_sim/native/64
+      - qemu_x86
+      - qemu_cortex_m3
```

Comment:
Twister documentation dissuades using `platform_allow`. Do we need it?

```suggestion
```

Reply:
Probably not. I added them as they were present in another test